config/action.d/htaccess.conf

```diff
@@ -0,0 +1,66 @@
+# Fail2Ban configuration file
+#
+# Author: Denis Pitzalis
+#
+# $Revision$
+#
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart =
+
+
+# Option:  actionstop
+# Notes.:  command executed once at the end of Fail2Ban
+# Values:  CMD
+#
+actionstop =
+
+
+# Option:  actioncheck
+# Notes.:  command executed once before each actionban command
+# Values:  CMD
+#
+actioncheck =
+
+
+# Option:  actionban
+# Notes.:  command executed when banning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    <ip>  IP address
+#          <file>  htaccess
+# Values:  CMD
+# You can have multiple virtual hosts
+#
+actionban = IP=<ip> &&
+	printf %%b "Deny from $IP\n" >> <htaccess1>
+#	printf %%b "Deny from $IP\n" >> <htaccess2>
+#	printf %%b "Deny from $IP\n" >> <htaccess3>
+
+
+# Option:  actionunban
+# Notes.:  command executed when unbanning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    <ip>  IP address
+#          <failures>  number of failures
+#          <time>  unix timestamp of the ban time
+# Values:  CMD
+#
+actionunban = IP=<ip> &&
+	sed -i "/Deny from $IP/d" <htaccess1>
```

Comment:
and not sure if worthwhile creating `IP=<ip>` -- just plug `<ip>` within sed line, and then can use `'` and thus a single `$` for anchoring